packages/core/src/pagination/pagination-button.element.scss

```diff
@@ -0,0 +1,58 @@
+@import './../styles/tokens/generated/index';
+
+:host {
+  --background: none;
+  --padding: 0;
+  --border-color: var(--color);
+  --border-width: 0;
+  --border-radius: #{$cds-alias-object-border-radius-100};
+  --box-shadow: 0 #{$cds-global-space-2} 0 0 #{$cds-alias-object-opacity-100} inset;
+  --disabled-background: none;
+  --disabled-color: #{$cds-alias-status-disabled};
+  --min-height: #{$cds-global-space-9};
+  --min-width: #{$cds-global-space-9};
+  color: var(--color);
+}
+
+.private-host {
+  --icon-color: var(--color);
+  color: var(--color);
+  background: var(--background);
+  min-width: var(--min-width);
+  min-height: var(--min-height);
+  border-color: var(--border-color);
+  border-radius: var(--border-radius);
+  border-style: solid;
+  border-width: var(--border-width);
+  font-size: var(--font-size);
+}
+
+:host(:active) .private-host {
+  box-shadow: var(--box-shadow);
+}
+
+:host(:hover),
+:host(:focus),
+:host(:active) {
+  .private-host {
+    filter: brightness(90%);
```

Comment:
We had talked about tokenizing values like this with percentage in them because the same percentage might not work for all colors when you apply a custom theme on core components, etc.

Do we have any examples of this @coryrylan? I forgot the context / component in which we had this discussion before.

Reply:
As we didn't need the entire base button style, I've only included this part from the base button style. 

And I agree. I also noticed it works differently for bright and dark colors.

Reply:
Yeah we talked about building out a cds-action-button component for all the common action style button that datagrid uses (filtering, multi-action). I think this works for now and once the action button is complete it could inherit it without any breaking api change to the pagination components.